doc/kubeletConfig-remediations.md

```diff
@@ -11,41 +11,45 @@ We want to accomplish:
 ## Add CaC `KubletConfig` remediation content 
 In CaC content, we are going to add `KubletConfig` remediation besides `MachineConfig` that we currently have.
 
-We will have remdiation that has object payload to be `KubletConfig`. As a exmaple below:
+We will have remediation that has object payload to be `KubletConfig`. As an example below:
```

Comment:
It's `KubeletConfig`, not `KubletConfig`

Reply:
Sorry for the typo